google/api_core/client_logging.py

```diff
@@ -0,0 +1,141 @@
+import logging
+import json
+import os
+
+from typing import List, Optional
+
+_LOGGING_INITIALIZED = False
+_BASE_LOGGER_NAME = "google"
+
+# Fields to be included in the StructuredLogFormatter.
+#
+# TODO(https://github.com/googleapis/python-api-core/issues/761): Update this list to support additional logging fields.
+_recognized_logging_fields = [
+    "httpRequest",
+    "rpcName",
+    "serviceName",
+]  # Additional fields to be Logged.
```

Comment:
```suggestion
    "credentialsType",
    "credentialsScope",
    "useSelfSignedJWT",
    "universeDomain",
]  # Additional fields to be Logged.
```
Add credentials related fields so that we can include information similar to the ruby library

```
I, [2024-09-30T16:52:15.809067 #1917313]  INFO -- google-cloud-secret_manager-v1: Created client for google.cloud.secretmanager.v1.SecretManagerService -- {"clientId":3220,"system":"google-cloud-secret_manager-v1","serviceName":"google.cloud.secretmanager.v1.SecretManagerService","credentialsId":3260,"credentialsType":"Google::Auth::ServiceAccountCredentials","credentialsScope":["https://www.googleapis.com/auth/cloud-platform"],"useSelfSignedJWT":false,"universeDomain":"googleapis.com"}
```